RELEASES.md

```diff
@@ -0,0 +1,248 @@
+| [0.1](https://github.com/containerd/containerd/releases/tag/v0.1.0) | End of Life | Mar 21, 2016 | - |
+| [0.2](https://github.com/containerd/containerd/tree/v0.2.x)         | Active | Apr 21, 2016      | Upon 1.0 release |
+| 1.0     | Next   | TBD  | TBD+1 year |
+
+Note that branches and release from before 1.0 may not follow these rules.
+
+This table should be updated as part of the release preparation process.
+
+### Backporting
+
+Backports in containerd are community driven. As maintainers, we'll try to
+ensure that sensible bugfixes make it into _active_ release, but our main focus
+will be features for the next _minor_. For the most part, this process is
+straightforward and we are here to help make it as smooth as possible.
+
+If there are important fixes that need to be backported, please let use know in
+one of three ways:
+
+1. Open an issue.
+2. Open a PR with cherry-picked change from master.
+3. Open a PR with a ported fix.
+
+__If you are reporting a security issue, please follow the responsible
+disclosure guidelines and reach out discretely__. Remember that backported PRs
+must follow the versioning guidelines from this document.
+
+Any release that is "active" can accept backports. Opening a backport PR is
+fairly straightforward. The steps differ depending on whether you are pulling
+a fix from master or need to draft a new commit specific to a particular
+branch.
+
+To cherry pick a straightforward commit from master, simply use the cherry pick
+process:
+
+1. Pick the branch to which you want backported, usually in the format
+   `release/<minor>.<major>`. The following will create a branch you can
+   use to open a PR:
+
+	```console
+	$ git checkout -b my-backport-branch release/<major>.<minor>.
+	```
+
+2. Find the commit you want backported.
+3. Apply it to the release branch:
+
+	```console
+	$ git cherry-pick -xsS <commit>
+	```
+4. Push the branch and open up a PR against the _release branch_:
+
+	```
+	$ git push -u stevvooe my-backport-branch
+	```
+
+   Make sure to replace `stevvooe` with whatever fork you are using to open
+   the PR. When you open the PR, make sure to switch `master` with whatever
+   release branch you are targeting with the fix.
+
+If there is no existing fix in master, you should first fix the bug in master,
+or ask us a maintainer or contributor to do it via an issue. Once that PR is
+completed, open a PR using the process above.
+
+Only when the bug is not seen in master and must be made for the specific
+release branch should you open a PR with new code.
+
+## Public API Stability
+
+The following table provides an overview the components covered by containerd
+versions:
+
+
+| Component  | Status   | Stablized Version | Links         |
+|------------|----------|-------------------|---------------|
+| GRPC API   | Alpha    | 1.0               | [api/](api) |
+| Go API     | Unstable | 1.1 tentative     | [godoc](https://godoc.org/github.com/containerd/containerd) |
```

Comment:
I think it's worth calling out somewhere that *because* the GRPC API is covered by the stability guarantee, applications built against the 1.0 Go API will continue to work with the same versions that the stable GRPC API will, as the Go API leverages the GRPC API.